back/src/Models/Task.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm";
+
+@Entity("tasks")
+export class Task {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  description: string;
+
+  @Column()
```

Comment:
צריך להגיד ל typeorm שזו עמודה מסוג date
```suggestion
  @Column('date')
```